pytorch_lightning/trainer/trainer.py

```diff
@@ -459,11 +459,7 @@ def train(self):
 
         try:
             # run all epochs
-            for epoch in range(self.current_epoch, self.max_epochs):
-
-                # reset train dataloader
-                if self.reload_dataloaders_every_epoch:
-                    self.reset_train_dataloader(model)
+            for epoch in range(start_epoch, end_epoch):
```

Comment:
this should be unchanged now
```suggestion
            for epoch in range(self.current_epoch, end_epoch):
```